integrated_channels/moodle/exporters/content_metadata.py

```diff
@@ -16,18 +16,27 @@ class MoodleContentMetadataExporter(ContentMetadataExporter):
         Moodle implementation of ContentMetadataExporter.
     """
     DATA_TRANSFORM_MAPPING = {
-        'fullname': 'title',
+        'fullname': 'fullname',
         'shortname': 'shortname',
         'idnumber': 'key',
-        'summary': 'description',
         'startdate': 'start',
         'enddate': 'end',
         'categoryid': 'categoryid',
+        'format': 'format',
+        'announcement': 'announcement',
     }
 
-    LONG_STRING_LIMIT = 1700  # Actual maximum value we can support for any individual course
+# default url length = 8000 bytes / max char size of 4 bytes = 2000 char limit
+# subtracted arbitrary char buffer for template HTML, token, and other query params we send.
+    LONG_STRING_LIMIT = 1500
     SKIP_KEY_IF_NONE = True
 
+    ANNOUNCEMENT_TEMPLATE = '<div><div style="display: inline-block">' \
```

Comment:
Is this mobile/responsive compatible? If not, does it need to be? If it is, what are the test criteria?

Reply:
Mobile compatibility was not part of requirements. I'm not sure how we'd test mobile specific display either tbh. Something I should probably take into account/learn in the future.